[Rules] Add Disable RUM to configuration rule settings (#14537)
cloudflare/cloudflare-docs: content/rules/configuration-rules/settings.md

{{</details>}}

## Disable Real User Monitoring (RUM)

[Cloudflare Web Analytics](/analytics/web-analytics/), also known as Real User Monitoring (RUM), is Cloudflare's free, privacy-first analytics for your website.

Use this setting to turn off Web Analytics for matching requests.

{{<Aside type="warning" header="Warning">}}

> Configuration rules have precedence over any [Web Analytics rules](/analytics/web-analytics/configuring-web-analytics/rules/). If you configure a rule in Web Analytics to enable the feature and a configuration rule to disable it, the configuration rule will win.

Reviewer: ```suggestion
Configuration rules have precedence over any [Web Analytics rules](/analytics/web-analytics/configuring-web-analytics/rules/). If a request matches a rule in Web Analytics but a configuration rule is in place to disable it, the configuration rule will win.
```

Reviewer: Not sure if it helps. Guess I was just a bit confused by the idea of configuring a rule to enable a feature.
